src/compiler/lsp/protocol_summary.cc

```diff
@@ -396,13 +396,17 @@ void Writer::print_class(ir::Class* klass) {
   safe_print_symbol(klass->name());
   print_range(klass->range());
   this->printf("%d\n", toplevel_ids_.at(klass));
-  const char* kind;
-  if (klass->is_interface()) {
-    kind = "interface";
-  } else if (klass->is_abstract()) {
-    kind = "abstract";
-  } else {
-    kind = "class";
+  const char* kind = "";  // Initialize with value to silence compiler warnings.
+  switch (klass->kind()) {
+    case ir::Class::CLASS:
+      kind = klass->is_abstract() ? "abstract" : "class";
```

Comment:
It is a bit funky that this doesn't print 'abstract class' for abstract classes, but just 'abstract'.

Reply:
fixed in https://github.com/toitlang/toit/pull/1863